### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Suvam713**
Please approve

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
require review

File: notebooks/Users/484307@mgmresorts.com/FeatureSE/EDH292_test.scala [hunk 1]
```diff
@@ -0,0 +1,108 @@
+// Databricks notebook source
+//read the source from RAW JSON layer
+val myDF=spark.read.format("json").load("/mnt/adlsedhdev/EDH/Timberline/Supplier/RawCDM/Data/9999-12-31")
+myDF.count
+
+// COMMAND ----------
```
Comment (Suvam713) [inline notebooks/Users/484307@mgmresorts.com/FeatureSE/EDH292_test.scala:6]: Changed as per requested

**Review by Suvam713 — COMMENTED** [review]
Please approve now

**Review by ghost — APPROVED** [review]
Approve now

**Pull request merged**